### PR TITLE
fix(alerts): Replace custom breadcrumbs w/ component

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/details/header.tsx [hunk 1]
```diff
@@ -80,20 +80,21 @@ export default class DetailsHeader extends React.Component<Props> {
...
-              <React.Fragment>
-                <IconChevron direction="right" color={theme.gray1} size={space(1.5)} />
-                <IncidentDate>{dateStarted}</IncidentDate>
-              </React.Fragment>
-            )}
-          </Breadcrumb>
+          <AlertBreadcrumbs
+            crumbs={[
+              {label: t('Alerts'), to: `/organizations/${params.orgId}/alerts/`},
+              {label: alertBreadcrumbLabel},
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/alerts/details/header.tsx:95]: You should provide custom `key` here as the `label` was usually used for automatic `key` generation, but now the `label` got changed from string to node.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/alerts/details/header.tsx:95]: Ah good point

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/alerts/details/header.tsx]: Fixed

**Comment by matejminar** [comment]
Should we remove styles of IncidentDate component (or just pass label as a string)? I kind of don't like that we are not consistent with styles of active item in breadcrumbs:

image

image

image

VS

image
(smaller text and different color)

**Review by iProgramStuff — APPROVED** [review]
Looks good to me, other than what Matej said about inconsistent styles 👍

**Comment by EvanPurkhiser** [comment]
Going to merge for now and revisit later.

**Pull request merged**